config.base.yaml

```diff
@@ -27,13 +27,16 @@ alerts:
   emailalert:
     plugin: cloudmarker.alerts.emailalert.EmailAlert
     params:
+      is_ssl_required: False
       host: smtp.example.com
       port: 25
       sender: CloudMarker Alert <no-reply@example.com>
       to:
         - recipient@example.com
       subject: Anomaly notification
       body: default text
+      username: username
+      password: password
```

Comment:
Let us get a little bit creative here to very clearly highlight the strings that need to be changed. My suggestions:

```
username: alice
password: hunter2
```

Reply:
https://github.com/cloudmarker/cloudmarker/commit/11ff34083ba27900832dc7daf05a72cd2a99caca#diff-abc06351b88de7752624acd35f48332cR40